lib/facter/rhsm_repos.rb

```diff
@@ -1,7 +1,7 @@
 Facter.add('rhsm_repos') do
   setcode do
     repos = Array.[]
-    repo_list = Facter::Core::Execution.exec('subscription-manager repos --list-enabled | grep "Repo ID" | cut -d " " -f 5')
+    repo_list = Facter::Core::Execution.exec('yum repolist | awk '/Red Hat/ {print $1}'')
```

Comment:
single quotes in single quotes doesn't work. Can you excahnge the outta quotes with double quotes? That should work.